django/applications/catmaid/control/nat.py

```diff
@@ -1627,7 +1628,7 @@ def dotprops_for_skeletons(project_id, skeleton_ids, omit_failures=False,
         # Make sure all temporary R values are garbage collected. With many
         # skeletons, this can otherwise become a memory problem quickly (the
         # Python GC doesn't now about the R memory).
-        del([r_nodes, r_connectors, r_tags, skeleton_data])
+        del r_nodes, r_connectors, r_tags, skeleton_data
```

Comment:
This syntax is easier to read and makes mypy happy

Reply:
I agree, looks nicer.